edit.go

```diff
@@ -56,12 +56,12 @@ type Cmd struct {
 
 type Cmdtab struct {
 	cmdc    rune                   // command character
-	text    byte                   // takes a textual argument?
-	regexp  byte                   // takes a regular expression?
-	addr    byte                   // takes an address (m or t)?
+	text    bool                   // takes a textual argument?
```

Comment:
Aside: how big is a bool in golang?

Reply:
1 byte but may be implementation dependent? Anyway, this is more about readability than saving space. 🙂

Reply:
Indeed. The change makes the code much easier to read.